openpype/hosts/maya/plugins/load/load_audio.py

```diff
@@ -76,7 +92,9 @@ def update(self, container, representation):
             project_name, version["parent"], fields=["parent"]
         )
         asset = get_asset_by_id(
-            project_name, subset["parent"], fields=["parent"]
+            project_name,
+            subset["parent"],
+            fields=["parent", "data.frameStart", "data.frameEnd"]
         )
 
         source_start = 1 - asset["data"]["frameStart"]
```

Comment:
I wonder, instead of setting the attributes manually. Would it also work to do this?

```python
cmds.sound(audio_node, edit=True, offset=start_frame)
```

Also, is it technically weird that we set `sourceStart` and `sourceEnd` on the sound node from the asset's data here but we don't do that on `load`? Doesn't that mean that technically the sound node behaves differently after `update` than on `load`?

I have a feeling that updating to the SAME version might even start showing different offsets?

Reply:
Since it's consistent with how  it was I feel like we can push it to a separate PR but we should definitely pursue consistent behavior between load and  updating the audio node.

Reply:
Would rather just work it into this PR cause I doubt I'll be doing another PR for this.

We can also just leave it to an issue (that wont be resolved :) )